app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -1,7 +1,7 @@
 .l-header {
-  background: $color-green-primary;
```

Comment:
The media query fix is better I think because of the px/rem issue for the header height means there are potentially *slight* variations in the depth of the two elements.